tests/quality/README.md

```diff
@@ -0,0 +1,58 @@
+# DSPy Quality Tests
```

Comment:
Open to alternative names here. We could call this "Adapter" testing, but there's more being tested here than just Adapter logic - we're writing full programs and making quality assertions on their outputs.

I chose "quality" because one objective of DSPy is to provide consistent quality across LLMs